packages/actions/resources/views/select-action.blade.php

```diff
@@ -1,28 +1,31 @@
 @php
     $id = $getId();
     $isDisabled = $isDisabled();
+    $attributes = $getExtraAttributeBag();
```

Comment:
This is only used once, so there's no need to assign it to a variable. :)

Reply:
Oh, I thought that was a filament convention to assign them at the top. :D 

I will change it. However, I just realised that removing the input wrapper will not work.

The reason it was added as CSS is, that it uses the same `select-action` view as any other select action, to prevent duplicity.

So either we need to revert back to the CSS, or add the input wrapper conditionally.

Reply:
> Oh, I thought that was a filament convention to assign them at the top. :D

Generally only to prevent a getter from being called more than once.

> I will change it. However, I just realised that removing the input wrapper will not work.
> 
> The reason it was added as CSS is, that it uses the same `select-action` view as any other select action, to prevent duplicity. Now even the regular SelectActions (such as the LocaleSwitcher) won't have the input wrapper.
> 
> So either we need to revert back to the CSS, or add the input wrapper conditionally.

Hmm, the wrapper also handles the disabled state (background color).
What was the CSS for again?

Reply:
> Hmm, the wrapper also handles the disabled state (background color). What was the CSS for again?

If there's an input wrapper, it adds a background and a border to make it look like a standalone input. That works great when using as a Header Action (for example the Locale Switch).

However, since I added the SelectAction as a suffix action to form fields, they already have a background and a border.

So I had to remove it, which I did using the CSS from my first commit. That can also be done by simply removing the input wrapper, but the thing is:

the blade view is the same for both select actions, whether they are used as a header action or as a prefix/suffix action.

Maybe you have a better idea, but what comes to my mind currently is either using the CSS again or creating a method, say `hasInputWrapper()` which will return false for the form field SelectAction and then conditionally add/remove the input wrapper in the blade view.

Reply:
Thanks for clarifying, would have to think about this.